### ISSUE TITLE
SearchBar should require an aria-label (or aria-labelledby)

### ISSUE THREAD

**Issue by gabro** [buildo/bento-design-system#267]
`SearchBar` does not have a visible label, so it should require either `aria-label` or `aria-labelledby` so that it stays accessible.

We can use the same pattern used by `Checkbox` here (see the use of `O.AtLeast`, to make sure we get either one of the two props):

https://github.com/buildo/bento-design-system/blob/1456aaa50eb763c5eed4c6a41e8a6bc1905e3901/packages/bento-design-system/src/Checkbox/createCheckbox.tsx#L9
